locale/de/config.cfg

```diff
@@ -0,0 +1,87 @@
+[entity-name]
+gunship-airborne=Kanonenboot
+cargo-plane-airborne=Frachtflugzeug
+jet-airborne=Jet
+flying-fortress-airborne=Fliegende Festung
+better-cargo-plane-airborne=Besseres Frachtflugzeug
+even-better-cargo-plane-airborne=Noch besseres Frachtflugzeug
+
+[entity-description]
+gunship-airborne=Aufklärungs- und Kampfflugzeug
+cargo-plane-airborne=Eine fliegende Langstrecken-Truhe.
+jet-airborne=Wenn du SCHNELL an einen Ort gelangen musst.
+flying-fortress-airborne=Der ultimative Luftüberlegenheitsflieger.
+better-cargo-plane-airborne=Ein bessere fliegende Langstrecken-Truhe.
+even-better-cargo-plane-airborne=Ein noch bessere fliegende Langstrecken-Truhe.
+
+[mod-setting-name]
+aircraft-realism-turn-radius=Realistischer Wenderadius
+aircraft-realism-acceleration=Realistische Beschleunigung
+aircraft-realism-braking-speed=Realistische Bremskraft
+aircraft-realism-takeoff-health=Erhöhter Bodenkollisionsschaden
+aircraft-realism-no-aerial-shadows=Flugzeugschatten
+aircraft-realism-fuel-usage-multiplier-grounded=Kraftstoffverbrauchsmultiplikator (Boden)
+aircraft-realism-fuel-usage-multiplier-airborne=Kraftstoffverbrauchsmultiplikator (Luft)
+aircraft-realism-strict-runway-checking=Zwingende Rollbahn Vorhandenheit
+aircraft-realism-strict-runway-checking-maximum-tile-vehicle-friction=Modifikator für die maximale Fahrzeugreibung der Landebahn
+aircraft-realism-strict-runway-max-taxi-speed=Strikte maximale Geschwindigkeit für die Rollbahn
+aircraft-realism-environmental-impact=Tödliche Zusammenstöße mit der Umwelt.
+aircraft-emit-pollution=Flugzeuge geben Umweltverschmutzung ab
+aircraft-pollution-amount=Menge der abgegebenen Umweltverschmutzung
+aircraft-takeoff-speed-gunship=Kanonenboot Startgeschwindigkeit ---
+aircraft-takeoff-speed-cargo-plane=Frachtflugzeug Startgeschwindigkeit ---

```

Comment:
Remove the horizontal dash (---)
It only needs to exist for `aircraft-takeoff-speed-gunship` and `aircraft-landing-speed-gunship` as a separator